pyproject.toml

```
@@ -0,0 +1,80 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "py2spack"
+authors = [{name = "davhofer", email = "davhofer@ethz.ch"}]
+readme = "README.md"
+license = {text = "MIT"}
+classifiers = ["License :: OSI Approved :: MIT License"]
+dynamic = ["version", "description"]
```

Actually for hatchling the description should be static and stated, for example:
```suggestion
description = "Tool to create spack package definitions from distributable Python packages."
dynamic = ["version"]
```